app/components/ItemList.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from "react"
+import { Pane, ListGroup } from "react-photonkit"
+import Item from "./Item"
+
+const ItemList = ({
+  items,
+  handleItemClick
+}) => (
+  <Pane>
+    <ListGroup>
+      {items.map((todo) => {
+        return (
+          <Item
+            key={todo.id}
+            text={todo.text}
+            active={todo.active}
+            handleClick={() => { handleItemClick(todo.id) }}
```

Comment:
Creating functions inside the render method is usually an anti-pattern, as they'll be re-created on every render, putting unnecessary pressure on the garbage collector. (The [react/jsx-no-bind](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md) lint rule talks a bit about it.)